pkg/parser/parser.go

```diff
@@ -49,13 +51,15 @@ var supportedEcosystems map[string]bool = map[string]bool{
 	models.EcosystemCyDxSBOM:      true,
 	models.EcosystemSpdxSBOM:      true,
 	models.EcosystemGitHubActions: true,
+	models.EcoSystemTerraform:     true,
 }
 
 // TODO: Migrate these to graph parser
 var customExperimentalParsers map[string]lockfile.PackageDetailsParser = map[string]lockfile.PackageDetailsParser{
```

Comment:
Add the standard filename i.e. `.terraform.lock.hcl` to `dependencyGraphParsers` map so that this parser can automatically be discovered based on filename.

`parser.FindParser(..)` is used by other packages to see if a parser exists for a given file. `FindParser(..)` in turn looks up `dependencyGraphParsers` to find parsers for standard filenames.